### DISCUSSION TITLE
request for add new game

### DISCUSSION THREAD

**Discussion by posml** [Tsyuay/FrivReborn#57] [Ideas]
Hello! 

i wanted to know if it is possible to add this game called "Electric Man":
![image](https://user-images.githubusercontent.com/125620780/219525503-41b2528b-6605-4265-96f0-f98a3e1c283a.png)
(this one)

![image](https://user-images.githubusercontent.com/125620780/219525274-3aea8773-f245-4e16-98bd-b032ad60af21.png)
(gameplay)

keyboard buttons: ![image](https://user-images.githubusercontent.com/126031540/220780640-07197bc4-29e0-4f8d-aff9-9961e299cc0f.jpg)

Swf funo: https://web.archive.org/web/20200618103734/Http://l.funo.com/z/electricman/electricman.swf
walkthrought: https://web.archive.org/web/20201230235733/https://www.friv.com/x/walkthroughs/electricman.html
thanks for the attention!

**Comment by Katrina19kj** [comment]
Put the keyboard buttons.
![Screenshot_20230222-193241_WhatsApp](https://user-images.githubusercontent.com/126031540/220780640-07197bc4-29e0-4f8d-aff9-9961e299cc0f.jpg)

Use this link because the other one is "Electricman 2" use this link that it's the first electricman.
Swf funo: https://web.archive.org/web/20200618103734/Http://l.funo.com/z/electricman/electricman.swf

**Reply by posml** [reply]
thank you so much!
